### ISSUE TITLE
Catch database connection error for polling jobs

### ISSUE THREAD

**Issue by comaurice** [department-of-veterans-affairs/va.gov-team#95986]
## Value Statement

**_As a_** Dev

**_I want to_** be aware of any database connection error for polling jobs

**_So that_** we can prevent an uncaught polling error

---

## Background Context

- In Cron jobs, first call checks for matching criteria for records. NO rescue on first query, the job throws an error into WatchTower queue. 

[Line where error happens] (https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/sidekiq/decision_review/saved_claim_hlr_status_updater_job.rb#L22)

## Outcome, Success Measure, KPI(S), and Tracking Link

- Add here

## Acceptance Criteria

- [ ] Code update confirms notification of any caught errors
- [ ] _AC_
- [ ] _AC_

## Designs and Build Notes

- Add here

### Enablement team (if needed)
@ member of team

## Out of scope

- Add here

## Open questions

- Add here

## Tasks

- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by eileen-coforma** [comment]
Dependent on #96645 to merge before this can clos.
